### ISSUE TITLE
escape backtick violates MD038

### ISSUE THREAD

**Issue by unional** [DavidAnson/markdownlint#46]
Writing `` ` ``  is marked as violating MD038 `no-space-in-code`

```markdown
`` ` ``
```

**Comment by DavidAnson** [comment]
Per http://spec.commonmark.org/0.27/#code-spans your example is parsed as:

1. token: begin-inline-code
2. text: space backtick space
3. token: end-inline-code

At this point, `MD038` is correct in reporting a violation.

However, the spec also says that leading/trailing whitespace is to be removed from the inline - at which point `MD038` is wrong to report a violation.

I think this is what you're reporting?

**Comment by unional** [comment]
I think so. The use case is to render backtick in the inline code block.
